components/automate-ui/src/app/page-components/run-history/run-history.component.html

```diff
@@ -1,54 +1,72 @@
 <chef-click-outside omit="run-history-button" (clickOutside)="closeRunHistory()">
   <chef-side-panel id="run-history-panel" tabindex="0" [visible]="visible">
-    <div class="history-header">
-      <h2 class="history-heading"> Run History</h2>
-      <chef-button (click)="closeRunHistory()" secondary class="close-button" label="close">
+    <div class="side-panel-header">
+      <chef-icon class="header-icon">restore</chef-icon>
+      <div class="header-text">
+        <h4><strong>Run history for node:</strong></h4>
+        <p>{{ nodeName }}</p>
+      </div>
+      <chef-button secondary label="close" (click)="closeRunHistory()">
         <chef-icon>close</chef-icon>
       </chef-button>
     </div>
-    <div class="filter-selectors">
-      <div class="selector all" (click)="onStatusChange(selectedStatus.All)" [ngClass]="{active: selected == selectedStatus.All}">
-          <chef-icon>check_circle</chef-icon>
-          <div>{{stats(selectedStatus.All)}}</div>
-      </div>
-      <div class="selector failed" (click)="onStatusChange(selectedStatus.Failure)" [ngClass]="{active: selected == selectedStatus.Failure}">
-        <chef-icon>warning</chef-icon>
-        <div>{{stats(selectedStatus.Failure)}}</div>
-      </div>
-      <div class="selector successful" (click)="onStatusChange(selectedStatus.Success)" [ngClass]="{active: selected == selectedStatus.Success}">
-        <chef-icon>check_circle</chef-icon><div>{{stats(selectedStatus.Success)}}</div>
-      </div>
-      <div>
-        <app-date-selector (select)="dateSelected($event)" [initialDateTerm]="defaultSelectionTerm"> </app-date-selector>
-      </div>
-    </div>
-    <ul>
-      <li class="history-item" *ngFor="let history of nodeHistory" (click)="onSelect(history)">
-        <div class="arrow">
-          <chef-icon *ngIf="activeRunId === history.runId">chevron_left</chef-icon>
-        </div>
-        <div class="status" [ngClass]="history.status">
-          <chef-icon>{{ history.status | chefStatusIcon }}</chef-icon>
-        </div>
-        <div class="duration">
-          <p><strong>{{ history.startTime | datetime: RFC2822 }}</strong></p>
+
+    <div class="side-panel-body">
+      <div class="side-panel-body-header">
+        <div>
+          <p>Select a run below to view results on that date</p>
         </div>
-        <div class="date-time">
-          {{getDuration(history.startTime, history.endTime)}}
+
+        <div class="filter-selectors">
+          <button class="selector all" (click)="onStatusChange(selectedStatus.All)" [ngClass]="{active: selected == selectedStatus.All}">
```

Comment:
Opted to change these from `div` to `button` so that they are natively keyboard accessible

Reply:
thank you!!!!!